app/signals/apps/users/rest_framework/serializers/profile.py

```diff
@@ -27,7 +27,7 @@ class Meta:
             'updated_at',
         )
 
-    def get_departments(self, obj):
+    def get_departments(self, obj) -> list[str]:
```

Comment:
Does this work ok? I thought you had to import `List` a in `from typing import List`

Reply:
It works, `typing.List` is deprecated!